src/xunit.console.netcore/Program.cs

```diff
@@ -86,7 +88,9 @@ public static int Main(string[] args)
             }
             finally
             {
+#if !NETCORE
```

Comment:
Consider calling Console.ResetColor() here instead; that'll have the same effect, and is supported.